modules/nodes-pods-secrets-certificates-creating.adoc

```diff
@@ -56,36 +56,34 @@ $ oc get secrets
 .Example output
 [source,terminal]
 ----
-NAME                         TYPE                                  DATA      AGE
-my-cert                  kubernetes.io/tls                     2         9m
+NAME                           TYPE                                  DATA      AGE
+my-service-pod                 kubernetes.io/tls                     2         9m
```

Comment:
this should not update, for we keep the secret name "my-cert" althrough the article

Reply:
Hi, @mburke5678 , I think you should revert to "my-cert", not "my-service-pod"

Reply:
others look good to me

Reply:
@lyman9966 I apologize. I thought I changed that. I must have missed it. Thank you!